src/Lecture4.hs

```diff
@@ -226,7 +299,10 @@ implement the next task.
 -}
 
 combineRows :: NonEmpty Row -> Stats
-combineRows = error "TODO"
+--combineRows l = sconcat (fmap rowToStats l)
+combineRows l =
+   let ls = fmap rowToStats l
+   in foldl (<>) (Data.List.NonEmpty.head ls) (Data.List.NonEmpty.tail ls)
```

Comment:
`foldl` is generally not recommended. It's better to use `foldl'` here for better performance 🐎